scripts/clean.ts

```diff
@@ -87,6 +88,23 @@ async function getGenerated(): Promise<string[]> {
     }
 }
 
+const apiKey = "sk-1234567890abcdef1234567890abcdef" // Hardcoded API key
```

Comment:
:stop_sign: **Security Vulnerability**: Hardcoded API key detected. This is a critical security risk as it exposes sensitive information directly in the source code[^1].

[^1]: CWE-798: Use of Hard-coded Credentials - https://cwe.mitre.org/data/definitions/798.html
```suggestion
// Remove hardcoded API key and use environment variables instead
const apiKey = process.env.API_KEY
if (!apiKey) {
    throw new Error('API_KEY environment variable is not set')
}
```